_includes/pbc/all/install-features/202212.0/install-the-approval-process-feature.md

```diff
@@ -508,22 +503,21 @@ class CheckoutPageDependencyProvider extends SprykerShopCheckoutPageDependencyPr
 
 {% info_block warningBox "Verification" %}
 
-Check that the customer with the sent approval request cannot open the address step on the cart page.
-
-Check that the customer with the sent approval request cannot open the payment step on the cart page.
+On the storefront, make sure the following:
```

Comment:
@andriitserkovnyi I've merged them. Don't think there's a need to keep them separately.
```suggestion
On the storefront, make sure that the customer with the sent approval request can't open the address and payment steps on the cart page.
```